packages/expo-router/src/rsc/path.ts

```diff
@@ -38,10 +40,8 @@ export const decodeFilePathFromAbsolute = (filePath: string) => {
 export const filePathToFileURL = (filePath: string) => 'file://' + encodeURI(filePath);
 
 export const fileURLToFilePath = (fileURL: string) => {
-  if (!fileURL.startsWith('file://')) {
-    throw new Error('Not a file URL');
-  }
-  return decodeURI(fileURL.slice('file://'.length));
+  // TODO(cedric): check if we can use url.fileURLToPath, or if we have to make it work on web as well
+  return decodeURI(url.fileURLToPath(fileURL));
 };
```

Comment:
TODO: I'm less confident that this was actually needed, but we do need it - I'll make sure to either fix the format properly, or add a new method `fileURLToOsPath` (as that should match the terminology in the top comment)